tests/test_acm/test_acm.py

```diff
@@ -495,20 +493,21 @@ def test_request_certificate_no_san():
 
 
 # Also tests the SAN code
-@freeze_time("2012-01-01 12:00:00", as_arg=True)
 @mock_acm
-def test_request_certificate_issued_status(frozen_time):
+def test_request_certificate_issued_status():
     # After requesting a certificate, it should then auto-validate after 1 minute
     # Some sneaky programming for that ;-)
     client = boto3.client("acm", region_name="eu-central-1")
 
-    resp = client.request_certificate(
-        DomainName="google.com",
-        SubjectAlternativeNames=["google.com", "www.google.com", "mail.google.com"],
-    )
+    with freeze_time("2012-01-01 12:00:00"):
```

Comment:
Workaround as pytest doesn't seem to like the custom fixture argument ('frozen_time')